### ISSUE TITLE
Query parameter style not supported?

### ISSUE THREAD

**Issue by taqqanori** [anttiviljami/openapi-backend#82]
According to https://swagger.io/docs/specification/serialization/#query, query parameter value can be CSV like
```
/users?id=3,4,5
```

when definition is
```yaml
style: "form"
explode: false
```

But currently openapi-backend seems to accept only below even if the definition is above
```
/users?id=3&id=4&id=5
```

Everything else of openapi-backend is great and very suitable for my project. 
Thanks.

**Comment by anttiviljami** [comment]
Hi @taqqanori ! This is fixed in openapi-backend@3.4.0 !

**Comment by anttiviljami** [comment]
See test added for this: https://github.com/anttiviljami/openapi-backend/blob/master/src/router.test.ts#L168
